src/contrib/cluster/Akka.Cluster.Tools.Tests.MultiNode/Singleton/ClusterSingletonManagerChaos2Spec.cs

```diff
@@ -92,6 +92,7 @@ public void ClusterSingletonManager_in_chaotic_cluster_should_take_over_when_tre
             {
                 var memberProbe = CreateTestProbe();
                 Cluster.Subscribe(memberProbe.Ref, new[] { typeof(ClusterEvent.MemberUp) });
+                // TODO: could be racy here
                 memberProbe.ExpectMsg<ClusterEvent.CurrentClusterState>();
```

Comment:
Think that terminated message is for the probe? Possible that we have a race issue with the `TestProbe`? Or do you think this is an issue with `Cluster` ?